tictactoe.Tests/MockGameRulesInterface.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+namespace tictactoe.Tests
+{
+    public class MockGameRulesInterface : IGameRulesInterface
+    {
+        public bool DrawIsCalled;
+        public bool WonIsCalled;
+        public bool GameIsOver;
+
+        public bool Draw(Board board) => DrawIsCalled = true;
+
+        public bool Over(Board board)
+        {
+            GameRules rules = new GameRules();
+            if (rules.Over(board))
+            {
+                GameIsOver = true;
+                return GameIsOver;
+            }
+            GameIsOver = false;
+            return GameIsOver;
```

Comment:
While I like how descriptive the code is, it can also be simplified quite a bit. Can you find a way to reduce this method to a single line?

Reply:
commit a47b5e5dd8b786822707c2e20c8ff76d4f94b71c